install_dependencies.sh

```diff
@@ -23,19 +23,33 @@ sudo apt install -y mitmproxy \
         libjavascriptcoregtk-4.0-18 \
         libjavascriptcoregtk-4.0-dev \
         libwebkit2gtk-4.1 \
-        libwebkit2gtk-4.1-dev
+        libwebkit2gtk-4.1-dev \
+		cargo \
+		nodejs
 
+echo "----------------------"
 echo "System deps installed."
 echo "Installing rust..."
+echo "----------------------"
 
 # Install rust.
 curl https://sh.rustup.rs -sSf | sh -s -- -y
 source "$HOME/.cargo/env"
 
+echo "-----------------------------"
 echo "Rust installation complete..."
+echo "Installing NodeSource NodeJS..."
+echo "-------------------------------"
+#Install NodeSource NodeJS
+sudo su <<E0F 
+curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+exit
+E0F
 
 # Install openssl.
+echo "---------------------"
 echo "Installing openssl..."
+echo "---------------------"
 pushd /tmp
 wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.16_amd64.deb
```

Comment:
Remove the installation of volta now that nodesource is the install method of node